cinema/models.py

```diff
@@ -35,12 +40,19 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image(instance: "Movie", filename: str) -> pathlib.Path:
+    filename = (f"{slugify(instance.title)}-"
+                f"{uuid.uuid4()}{pathlib.Path(filename).suffix}")
+    return pathlib.Path("uploads/movies/") / pathlib.Path(filename)
+
+
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
     genres = models.ManyToManyField(Genre)
     actors = models.ManyToManyField(Actor)
+    image = models.ImageField(upload_to=movie_image, null=True)
```

Comment:
Upload to where? I think movie_image_file_path would be better and more logical.